test/helpers/rule_test_helper.dart

```diff
@@ -0,0 +1,115 @@
+import 'dart:io';
+
+import 'package:analyzer/dart/analysis/results.dart';
+import 'package:analyzer/dart/analysis/utilities.dart';
+import 'package:dart_code_metrics/src/models/issue.dart';
+import 'package:dart_code_metrics/src/models/severity.dart';
+import 'package:dart_code_metrics/src/obsoleted/models/internal_resolved_unit_result.dart';
+import 'package:test/test.dart';
+
+class RuleTestHelper {
+  static Future<InternalResolvedUnitResult> resolveFromFile(
+    String filePath,
+  ) async {
+    final path = File(filePath).absolute.path;
+    final sourceUrl = Uri.parse(path);
+
+    // ignore: deprecated_member_use
+    final parseResult = await resolveFile(path: path) as ResolvedUnitResult;
+
+    return InternalResolvedUnitResult(
+      sourceUrl,
+      parseResult.content!,
+      parseResult.unit!,
+    );
+  }
+
+  static void verifyInitialization({
+    required Iterable<Issue> issues,
+    required String ruleId,
+    required Severity severity,
+  }) {
+    expect(issues.every((issue) => issue.ruleId == ruleId), isTrue);
+    expect(issues.every((issue) => issue.severity == severity), isTrue);
+  }
+
+  static void verifyIssues({
```

Comment:
@incendial please add ignore `Long method` and `Long Parameter List`

Reply:
Why? No report from the analyzer whatsoever.